rtic-macros/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ For each category, *Added*, *Changed*, *Fixed* add new entries at the top!
 ### Added
 
 - RTIC v2 now works on stable.
+- Unstable support for RISC-V targets compatible with `riscv-slic`
```

Comment:
This PR predates stable support, just it will land after :P Rebase does not care though

```suggestion
- Unstable support for RISC-V targets compatible with `riscv-slic`
- RTIC v2 now works on stable.
```

Reply:
Swapped